packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -0,0 +1,225 @@
+      />);
+    let comboBoxRoot = wrapper.find('.ms-ComboBox');
+    let inputElement = comboBoxRoot.find('[role="combobox"]');
+
+    expect(inputElement.text()).equals('');
+  });
+
+  it('Renders a selected item in uncontrolled case', () => {
+    let wrapper = mount(
+      <ComboBox
+        label='testgroup'
+        defaultSelectedKey='1'
+        options={ DEFAULT_OPTIONS }
+      />);
+    let comboBoxRoot = wrapper.find('.ms-ComboBox');
+    let inputElement = comboBoxRoot.find('input');
+
+    expect(inputElement.props().value).equals('1');
+  });
+
+  it('Renders a selected item in controlled case', () => {
+    let wrapper = mount(
+      <ComboBox
+        label='testgroup'
+        selectedKey='1'
+        options={ DEFAULT_OPTIONS }
+      />);
+    let comboBoxRoot = wrapper.find('.ms-ComboBox');
+    let inputElement = comboBoxRoot.find('input');
+
+    expect(inputElement.props().value).equals('1');
+  });
+
+  it('Renders a default value with options', () => {
+    let wrapper = mount(
+      <ComboBox
+        label='testgroup'
+        value='1'
+        options={ DEFAULT_OPTIONS }
+      />);
+    let comboBoxRoot = wrapper.find('.ms-ComboBox');
+    let inputElement = comboBoxRoot.find('input');
+
+    expect(inputElement.props().value).equals('1');
+  });
+
+  it('Renders a default value with no options', () => {
+    let wrapper = mount(
+      <ComboBox
+        label='testgroup'
+        value='1'
+      />);
+    let comboBoxRoot = wrapper.find('.ms-ComboBox');
+    let inputElement = comboBoxRoot.find('input');
+
+    expect(inputElement.props().value).equals('1');
+  });
+
+  it('Can change items in uncontrolled case', () => {
+    let comboBoxRoot;
+
+    try {
+      let wrapper = mount(
+        <ComboBox
+          label='testgroup'
+          defaultSelectedKey='1'
+          options={ DEFAULT_OPTIONS }
+        />);
+      comboBoxRoot = wrapper.find('.ms-ComboBox');
+      let buttonElement = comboBoxRoot.find('button');
+      buttonElement.simulate('click');
+      let secondItemElement = wrapper.getDOMNode().ownerDocument.querySelector('.ms-ComboBox-item[data-index="1"]');
+      ReactTestUtils.Simulate.click(secondItemElement);
+    }
+    finally {
+      let inputElement = comboBoxRoot.find('input');
+      expect(inputElement.props().value).equals('2');
+    }
+  });
+
+// Text cases for AutoComplete: True, AllowFreeform: True
+  it('Can insert text in uncontrolled case with autoComplete and allowFreeform on', () => {
+    let comboBoxRoot;
+    let inputElement;
+    try {
+      let wrapper = mount(
+        <ComboBox
+          label='testgroup'
+          defaultSelectedKey='1'
+          options={ DEFAULT_OPTIONS2 }
+          autoComplete={ true }
+          allowFreeform ={ true }
+        />);
+      comboBoxRoot = wrapper.find('.ms-ComboBox');
+      inputElement = comboBoxRoot.find('input');
+      inputElement.simulate('change', { target: {value: 'f'} });
+    }
+    finally {

```

Comment:
This followed the way dropdown tests were working. I assumed it was originally intentionally done that way, but I'll do some testing to see if it can be removed